src/js/components/sub-header.tsx

```diff
@@ -1,13 +1,15 @@
 import * as classNames from 'classnames';
 import * as React from 'react';
-// import Icon = require('react-fontawesome');
-import { connect } from 'react-redux';
+import { connect, Dispatch } from 'react-redux';
+import Icon = require('react-fontawesome');
 
 import { FetchError, isFetchError } from '../modules/errors';
+import Modal, { ModalType } from '../modules/modal';
 import Projects, { Project } from '../modules/projects';
 import Selected from '../modules/selected';
 import User, { Team } from '../modules/user';
 import { StateTree } from '../reducers';
+import Avatar from './common/avatar';
```

Comment:
Components should be in their own import group. Looks like this was not done correctly previously.